conf/config-default.yaml

```diff
@@ -52,8 +52,8 @@ apisix:
                                    # which cache to use by name in the admin api
         memory_size: 50m           # The size of shared memory, it's used to store the cache index
         disk_size: 1G              # The size of disk, it's used to store the cache data
-        disk_path: "/tmp/disk_cache_one"  # The path to store the cache data
```

Comment:
Have you added test to verify if this change works?

Reply:
Tested in yaml repo: https://github.com/Yiyiyimu/lua-tinyyaml/blob/test/symbols/spec/map_spec.lua#L57-L64 Do we need tests also in this repo?

BTW, I fix the bug causing the CI failure in that PR but I'm not sure if there are any other bugs. Is there any method to run CI with un-released lua rocks? Or should I create a temporary rock for the test, and if everything goes correctly, then we make the new release?

Reply:
Err, actually I mean testing with `disk_path` only. We can check if the default generated conf has this field.

BTW, why not release the rock first?

Reply:
Got it. Test added.

I guess releasing new versions for tiny patches could be troublesome 😿 If it's not of course it would be great